server.go

```diff
@@ -33,11 +35,12 @@ type Server struct {
 }
 
 func NewServer(port int) (server *Server, err error) {
-	server = &Server{}
-
-	server.Id = fmt.Sprintf("server:%d", port)
-	server.Port = port
-	server.BufferSize = 32768
+	server = &Server{
+		Id:         fmt.Sprintf("server:%d", port),
+		Port:       port,
+		BufferSize: 32768,
+		Timeout:    500 * time.Millisecond,
```

Comment:
This could be a bit higher to be on the safe side.

Reply:
This value affects sets deadlines *only* for TLS. Let's change to three secs.